### ISSUE TITLE
🐞 Can't Import Templates when the option to "Enable Unfiltered File Uploads" in Elementor Settings is disabled - [ED-13682]

### ISSUE THREAD

**Issue by viriava** [elementor/elementor#25071]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Uncaught (in promise) ReferenceError: elementor is not defined
    at Function.getUnfilteredFilesNotEnabledImportTemplateDialog (admin.min.js?ver=3.19.2:3:7750)
    at _callee$ (admin.min.js?ver=3.19.2:3:48690)
    at tryCatch (admin.min.js?ver=3.19.2:3:29344)
    at Generator.<anonymous> (admin.min.js?ver=3.19.2:3:30993)
    at Generator.next (admin.min.js?ver=3.19.2:3:29893)
    at asyncGeneratorStep (admin.min.js?ver=3.19.2:3:24778)
    at _next (admin.min.js?ver=3.19.2:3:25011)
    at admin.min.js?ver=3.19.2:3:25127
    at new Promise (<anonymous>)
    at HTMLFormElement.<anonymous> (admin.min.js?ver=3.19.2:3:24947)

<img width="889" alt="Screenshot 2024-02-08 at 8 29 04" src="https://github.com/elementor/elementor/assets/5452240/9f40798c-b623-44a3-8b77-635dcb9c6316">

<img width="807" alt="Screenshot 2024-02-08 at 8 48 15" src="https://github.com/elementor/elementor/assets/5452240/60611dc3-9107-4608-bc47-648606f38c64">

[Archive_5.json](https://github.com/elementor/elementor/files/14205694/Archive_5.json)

### Steps to reproduce

1. Disable "Enable Unfiltered File Uploads" in the Elementor->Settings->Advanced
2. Go to /templates -> import templates
3. Select file
4. Click import - see JS error

### Expected behavior

Import should work

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [ ] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

```txt
Latest WordPress
CLOUDWAYS Hosting
PHP 8.1
Latest Elementor and Elementor Pro
```

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by tkfv** [comment]
I have the exact same issue: Elementor/Pro both at 3.19.2, no other plugins installed, default theme.

**Comment by nicholaszein** [comment]
Hello @viriava.

We appreciate your involvement in our community. For efficient troubleshooting, please remember to include the **<br /><ins>FULL, and essential</ins> [System Info](https://elementor.com/help/hare-my-system-info/)** with each issue reported. We'll need it to help you effectively. 

#### ⚠️ As a gentle reminder, issues without it may need to be closed without troubleshooting. Providing this information upfront aligns with our guidelines, saves time, and ensures swift resolution. Thank you for understanding.

**Comment by RCady** [comment]
Hey all, I'm also running into this issue when attempting to import a kit library with Elementor Pro. It looks like the window variable of `elementor` is not defined, but is expected to be defined.

**Comment by KingGrizzly** [comment]
While import issues are being investigated/fixed, please include https://github.com/elementor/elementor/issues/24111 since it is also a bug impacting all users on import. Thanks.

**Comment by mpolek** [comment]
My solution is to change the two places in elementor/assets/js/admin.js and admin.min.js where it has:

if (!elementor.config.user.is_administrator) {

to:

if (!elementorAdmin.config.user.is_administrator) {

It's basically just a typo and easily fixable.

**Comment by nicholaszein** [comment]
Hello!

This issue was solved in version 3.19.3.

Feel free to update!

Best regards
